<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@804.vba10a18a1476">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2648.va9433432b33c">
    <parentIds>
      <string>61</string>
    </parentIds>
    <id>62</id>
    <startId>49</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1108.v57edf648f5d4">
      <startTime>1641410365881</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1108.v57edf648f5d4">
      <error class="hudson.AbortException">
        <detailMessage>docker login failed</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.docker.commons.impl.RegistryKeyMaterialFactory.materialize(RegistryKeyMaterialFactory.java:102)</trace>
          <trace>org.jenkinsci.plugins.docker.workflow.AbstractEndpointStepExecution2.doStart(AbstractEndpointStepExecution2.java:53)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.GeneralNonBlockingStepExecution.lambda$run$0(GeneralNonBlockingStepExecution.java:77)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>